cmd/edit/machinepool/cmd.go

```diff
@@ -132,6 +135,17 @@ func init() {
 			"This list will overwrite any modifications made to node tuning configs on an ongoing basis.",
 	)
 
+	flags.StringVar(&args.nodeDrainGracePeriod,
+		"node-drain-grace-period",
+		"",
+		"You may set a grace period for how long Pod Disruption Budget-protected workloads will be "+
+			"respected when the NodePool is replaced or upgraded.\nAfter this grace period, all remaining workloads "+
+			"will be forcibly evicted.\n"+
+			"Valid value is from 0 to 1 week (10080 minutes), and the supported units are 'minute|minutes' or "+
```

Comment:
@philipwu08 Are there any plans to support `day|days|week|weeks` as units?

Reply:
@robpblake Not at the moment. The reason is to be consistent with our existing node drain grace period on the cluster, which supports minutes and hours only.

Reply:
Thanks for the explanation @philipwu08